### DISCUSSION TITLE
Navigate to a subpage but display another domain?

### DISCUSSION THREAD

**Discussion by lkssmnt** [swup/swup#644] [Help] [answered]
Hi,

for a project I have two websites that should be switched between seamlessly using swup transitions. Lets call them website-a.com and website-b.com.

Website B is located under a subdomain of website A, but it also has it's own domain (website-b.com).

Navigating to website A and then clicking on a link that goes to website-a.com/website-b would be possible, but is it also possible to change the displayed domain in the browser, so that it displays website-b.com?

**Comment by daun** [comment, marked as answer]
No. As far as I know, the browser history api that's used for setting the new url doesn't support cross-domain access for security reasons.

My solution for these cases is a kind of fake "out" animation, followed by a normal browser visit with full reload and a fake "in" transition.

**Reply by lkssmnt** [reply]
Thank you! I figured as much. :)
